css/properties/background-size.json

```
@@ -134,11 +134,11 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "≤72"
+                "version_added": "3.6"
               },
               "firefox_android": "mirror",
               "ie": {
-                "version_added": "≤11"
+                "version_added": "9"
```

Let's revert this, because I have no evidence to support this (even if it's likely true):

```suggestion
                "version_added": "≤11"
```